flex-templates/python/modules/requirements.txt

```diff
@@ -1,2 +1,2 @@
 apache-beam==2.30.0
-google-cloud-dlp==1.0.0
+google-cloud-dlp==3.6.2
```

Comment:
are there changes needed in the open Java PR that we should consider wrt DLP library version?

Reply:
I'll talk to @vfigueiredo-cit regarding this and I'll let you know!